pom.xml

```diff
@@ -46,6 +46,12 @@
     			<version>2.23.1</version>
     			<scope>test</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.projectlombok</groupId>
+				<artifactId>lombok</artifactId>
+				<version>1.18.32</version>
+				<scope>provided</scope>
```

Comment:
Why scope provided?

Reply:
That's how I saw it recommeded in the lombok webpage. I read that there is also the "compile" scope, would "compile" be better?

Reply:
Do I keep it with the scope provided?

Reply:
ok, keep provided, if you later need lombok in the script module, just declare it.